### ISSUE TITLE
System.out initialization

### ISSUE THREAD

**Issue by rodykersten** [jayhorn/jayhorn#138]
Whenever the program contains a print statement, JayHorn says UNSAFE because it thinks that e.g. System.out is not initialized, see SatPrint. Should be an easy fix.

**Comment by martinschaef** [comment]
Makes sense ... I'd say let's focus on getting that parser for the IL running. Did you receive [my gist with the bnfc grammar](https://gist.github.com/martinschaef/85e24ed102bbcd757135043754b7245c)? Once we have that parser, we can write all this stuff in a prelude. Then we could also remove the special handling of the String array in main and make that part of the prelude ... much cleaner at least.

**Comment by rodykersten** [comment]
That sounds good, much cleaner than coding it indeed. Nice work on the grammar!

**Comment by martinschaef** [comment]
I downgraded that because technically, it's not a bug. Its just under-specified.
